README.md

```diff
@@ -12,9 +12,9 @@
 # Quick Links
 
 * [Installation](https://docs.tiledb.com/developer/installation/quick-install)
-* [Quickstart](https://docs.tiledb.com/developer/quickstart)
-* [API reference](https://tiledb-inc-tiledb-py.readthedocs-hosted.com/en/stable)
-* [Full developer documentation for all APIs and integrations](https://docs.tiledb.com/developer)
+* [TileDB Documentation](https://docs.tiledb.com/main/)
+* [Python API reference](https://tiledb-inc-tiledb-py.readthedocs-hosted.com/en/stable)
+* [Full developer documentation for all APIs and integrations](https://docs.tiledb.com/main/solutions/tiledb-embedded/api-usage)
```

Comment:
The text here should be `Python API Usage` for that link.

Reply:
The 'title page' linked is for all languages though so the text is correct.  I even copied that already (not committed though).

Reply:
Ah, that is true. Please ignore.